File: java_backup/my java/sayak/sum.java

public class sum
{
public static void main(int n)
{
    int a=0,b=1,c=2;
    System.out.println(a+","+b+","+c);
    for(int i=1;i<=(n-3);i++)
    {
        int d=(a+b+c);
        System.out.println(d);
        a=b;b=c;
        c=d;
    }
}
}

File: python/coursera_python/MICHIGAN/WEB/assign2/sum.py
# To scrap off data and find the sum of the numbers present in that data

import re


hand = open('a.txt')
list_=[]
sum=0
for line in hand:
	line = line.rstrip()
	
	y = re.findall('[0-9]+',line)
	#print(y)	# prints the list
	for i in y:
		sum=sum+int(i)
#print(list_)
print(sum)
	

